framework_tvb/tvb/core/entities/storage/operation_dao.py

```diff
@@ -281,7 +281,7 @@ def get_operations_for_datatype(self, datatype_gid, only_relevant=True, only_in_
         """
         try:
             query = self.session.query(Operation).filter(
-                                Operation.parameters.like('%' + datatype_gid + '%')).join(
+                                Operation.view_model_gid.like('%' + datatype_gid + '%')).join(
```

Comment:
Same here, the query becomes useless with this change.
We should review its upper usages, and try to replace the logic.